quickstarts/redis-prometheus/config.yml

```diff
@@ -36,7 +37,7 @@ description: |+
   If you are supporting Redis and Prometheus in a Kubernetes cluster, you can easily export Redis metrics to Prometheus using the [Prometheus Redis Exporter](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-redis-exporter).
 
 summary: |
-  New Relic provides crucial tools for monitoring your Redis instances using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system.
+  New Relic provides crucial tools for monitoring your Redis instances using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system. This quickstart is based on metrics sent to New Relic using Prometheus Agent or Prometheus Server.
```

Comment:
We need to keep the summary under 500 characterts.
Also to note, there is _no_ support for markdown inside our summary fields, so i would recommend removing that from the config

Reply:
![Screen Shot 2022-11-29 at 10 18 05 AM](https://user-images.githubusercontent.com/56952981/204613393-cf3a0ea4-c338-4af8-97ba-85a9c299a91d.png)